src/main/java/com/artipie/rpm/meta/XmlPrimary.java

```diff
@@ -438,15 +431,31 @@ public Format headerRange(final int start, final int end) throws XMLStreamExcept
 
         /**
          * Add list of providers.
-         * @param providers Provider names
+         * @param name Provides names
```

Comment:
@olenagerasimova This documentation should be fixed: the tag name is `provides`  , not `providers`, so this is not a list of providers, but a list of libraries provided by the package being represented by this `primary.xml` file.